### DISCUSSION TITLE
outline destinations not being saved

### DISCUSSION THREAD

**Discussion by mharol** [py-pdf/pypdf#1446] [Q&A] [answered]
When i open a pdf generated from pydf2 the outlines work fine.  As soon as I make any changes to the pdf the outlines remain however there are no destinations.  I didn't see where others are having this problem.  I would appreciate any feedback.
thanks
mike

**Comment by pubpub-zz** [comment, marked as answer]
This should be raised as a (possible) issue providing sample code instead of a discussion

**Reply by mharol** [reply]
thanks i submitted as issue
